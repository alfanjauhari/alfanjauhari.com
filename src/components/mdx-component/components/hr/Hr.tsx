import { HTMLAttributes } from 'react';

export type HrProps = HTMLAttributes<HTMLHRElement>;

export function Hr(props: HrProps) {
  return <hr className="border-gray-400" {...props} />;
}
