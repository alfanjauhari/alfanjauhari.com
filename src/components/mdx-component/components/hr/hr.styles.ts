import { styled } from '@/theme';

export const StyledHr = styled('hr', {
  borderTop: 0,
  borderBottom: '1px solid $colors$gray2',
  my: '$5',
});
