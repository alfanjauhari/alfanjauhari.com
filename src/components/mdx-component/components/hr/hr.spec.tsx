import { render, screen } from '@testing-library/react';
import { Hr } from './hr';

describe('MDXComponent/Hr', () => {
  it('should render successfuly', () => {
    render(<Hr data-testid="hr-component" />);

    expect(screen.getByTestId('hr-component')).toBeVisible();
  });

  it('should match snapshot', () => {
    render(<Hr data-testid="hr-component" />);

    expect(screen.getByTestId('hr-component')).toMatchSnapshot();
  });
});
