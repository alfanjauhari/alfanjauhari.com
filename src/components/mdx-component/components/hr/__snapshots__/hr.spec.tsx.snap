// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MDXComponent/Hr should match snapshot 1`] = `
<hr
  class="c-cPXaNr"
  data-testid="hr-component"
/>
`;
